operating_unit/models/res_users.py

```diff
@@ -14,6 +19,16 @@ def operating_unit_default_get(self, uid2=False):
         if not uid2:
             uid2 = self.env.user.id
         user = self.env["res.users"].browse(uid2)
+        # Check user read access on operating units.
+        # For other modules' (that even don't depend on ``operating_unit``) tests,
+        # we can have an AccessError raised for users without access to OUs.
+        # Therefore, we catch this exception and safely ignore it.
+        try:
+            self.env["operating.unit"].with_user(user).check_access_rights("read")
+            self.env["operating.unit"].with_user(user).check_access_rule("read")
+        except AccessError:
+            _logger.debug("User '%s' has no read access on operating units.", user.name)
```

Comment:
It might be better to log it as a warning, as debug log level might not be activated
```suggestion
            _logger.warning("User '%s' has no read access on operating units.", user.name)
```
Although the OCA CI might not be happy about it